.circleci/config.yml

```diff
@@ -246,6 +246,15 @@ jobs:
           name: Copy gradle config
           command: mkdir -p ~/.gradle && cp .circleci/gradle.properties ~/.gradle/gradle.properties
 
+      - run:
+          name: Set up secrets.properties for Maps frameworks
+          command: |
+            if [[ -n "$GOOGLE_MAPS_API_KEY" ]]; then \
+              echo "GOOGLE_MAPS_API_KEY=$GOOGLE_MAPS_API_KEY" >> secrets.properties
+              echo "MAPBOX_ACCESS_TOKEN=$MAPBOX_ACCESS_TOKEN" >> secrets.properties
```

Comment:
Feels like it would be nicer to have separate conditionals for each of these but I suppose that for our usage it doesn't matter. A fork could presumably want to have only the Mapbox keys.